test/system/roles/public_health/public_health_import_test.rb

```diff
@@ -11,150 +11,167 @@ class PublicHealthImportExportTest < ApplicationSystemTestCase
   end
 
-  test 'import sara alert format to isolation and validate file format' do
-    @@public_health_test_helper.import_sara_alert_format('locals2c4_epi', :isolation, 'Invalid-Excel-File.xlsx', :invalid_format, nil)
+  test 'import epi-x to isolation and validate fields' do
+    @@public_health_test_helper.import('locals1c2_epi', :isolation, :epix, 'Epi-X-Format-Invalid-Fields.csv', :invalid_fields, nil)
   end
 
-  test 'import sara alert format to exposure and validate headers' do
-    @@public_health_test_helper.import_sara_alert_format('state1_epi', :exposure, 'Sara-Alert-Format-Invalid-Headers.xlsx', :invalid_headers, nil)
+  # SDX Format
+
+  test 'import sdx to exposure and accept all' do
+    @@public_health_test_helper.import('state1_epi', :exposure, :sdx, 'SDX-Format.csv', :valid, nil)
   end
 
-  test 'import sara alert format to isolation and validate empty monitorees list' do
-    @@public_health_test_helper.import_sara_alert_format('state1_epi_enroller', :isolation, 'Sara-Alert-Format-Invalid-Monitorees.xlsx',
-                                                         :invalid_monitorees, nil)
+  test 'import sdx to isolation and accept some' do
+    @@public_health_test_helper.import('state2_epi', :isolation, :sdx, 'SDX-Format.csv', :valid, [1, 4, 5])
   end
 
-  test 'import sara alert format to exposure and validate fields' do
-    @@public_health_test_helper.import_sara_alert_format('state1_epi', :exposure, 'Sara-Alert-Format-Invalid-Fields.xlsx', :invalid_fields, nil)
+  test 'import sdx to exposure and validate file type' do
+    @@public_health_test_helper.import('locals2c4_epi', :exposure, :sdx, 'Invalid-Text-File.txt', :invalid_file, nil)
   end
 
-  test 'import sara alert format to isolation and validate fields' do
-    @@public_health_test_helper.import_sara_alert_format('state2_epi', :isolation, 'Sara-Alert-Format-Invalid-Fields.xlsx', :invalid_fields, nil)
+  test 'import sdx to isolation and validate file format' do
+    @@public_health_test_helper.import('locals2c3_epi', :isolation, :sdx, 'Invalid-Csv-File.csv', :invalid_format, nil)
   end
 
-  test 'import sara alert format to exposure and validate jurisdiction path' do
-    @@public_health_test_helper.import_sara_alert_format('locals2c4_epi', :exposure, 'Sara-Alert-Format-With-Jurisdictions-EW.xlsx', :invalid_fields, [])
+  test 'import sdx to isolation and validate headers' do
+    @@public_health_test_helper.import('locals2c4_epi', :isolation, :sdx, 'SDX-Format-Invalid-Headers.csv', :invalid_headers, nil)
+  end
+
+  test 'import sdx to exposure and validate empty monitorees list' do
+    @@public_health_test_helper.import('locals2c3_epi', :exposure, :sdx, 'SDX-Format-Invalid-Monitorees.csv', :invalid_monitorees, nil)
   end
 
-  test 'import sara alert format and cancel' do
+  test 'import sdx to exposure and validate fields' do
+    @@public_health_test_helper.import('locals1c2_epi', :exposure, :sdx, 'SDX-Format-Invalid-Fields.csv', :invalid_fields, nil)
```

Comment:
looks like the test failure in here isn't intermittent, I've rerun the tests a few times and it keeps failing

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1170/commits/0608463add7b71268c7debfd216a8b650d8d02af